LambdaEngine/Include/Game/ECS/Systems/Networking/ClientSystem.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "ECS/System.h"
+
+#include "Game/ECS/Components/Misc/Components.h"
+#include "Game/ECS/Components/Networking/NetworkComponent.h"
+
+#include "Networking/API/PlatformNetworkUtils.h"
+
+#include "Containers/CCBuffer.h"
+#include "Containers/TArray.h"
+
+namespace LambdaEngine
+{
+	struct GameState
+	{
+		int32 SimulationTick = -1;
+		glm::vec3 Position;
+		int8 DeltaForward = 0;
+		int8 DeltaLeft = 0;
+	};
+
+	class ClientSystem : public System, protected IClientHandler
+	{
+		friend class EngineLoop;
+
+	public:
+		DECL_UNIQUE_CLASS(ClientSystem);
+		virtual ~ClientSystem();
+
+		bool Connect(IPAddress* address);
```

Comment:
```suggestion
		bool Connect(IPAddress* pAddress);
```

Reply:
This error was fixed in an already merged PR